src/Components/Form_register/Form_Register.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react'
+import './Form_Reg.css'
+class Form_Register extends React.Component{
+
+    constructor(props) {
+        super(props)
```

Comment:
Trong `constructor` không hề dùng đến biến `props` do vậy hãy sửa thành
 ```suggestion
    constructor() {
        super()
```